Rakefile

```diff
@@ -120,7 +120,7 @@ end
 desc 'Run all code generation tasks'
 task :generate do
   # See note in BuildTools/.sourcery.yml for why we call without arguments
-  run_package_plugin(cmd: 'sourcery-command')
+  run_package_plugin(cmd: 'sourcery-command --disableCache')
```

Comment:
I added `--disableCache` to ensure every run I made locally to verify the setup run from scratch. I decided to keep it because, given we manually run this instead of having it as a build phase, having a guaranteed fresh output seems safer.